docs/src/components/mui/Button.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button as MuiButton, ButtonProps } from '@mui/material';
```

Comment:
Maybe for the chip, we really need to override is the Chip component in MUI https://mui.com/material-ui/react-chip/

Reply:
I added this story because I was checking the MUI button to see how it behaved when I changed the base style overrides. I am going to leave this story as it can be useful to see how the basic MUI button is rendered in the theme, but I will remove some of the overrides causing issues in the existing button.